### DISCUSSION TITLE
"La Chasse"

### DISCUSSION THREAD

**Discussion by thatmichaelpark** [probonopd/MiniDexed#324] [Show and tell]
Here's me playing a piece with several synths, one of which is MiniDexed: [https://youtu.be/3Clom3Ck2t4](https://youtu.be/3Clom3Ck2t4)

(I hope this is sufficiently on-topic.)

**Comment by probonopd** [comment]
We are always very interested in seeing and hearing what users are doing with MiniDexed.
Thanks for sharing!
